ccd-definition/AuthorisationCaseEvent/AuthorisationCaseEvent.json

```diff
@@ -978,5 +978,68 @@
     "CaseEventID": "addCaseNote",
     "UserRole": "caseworker-publiclaw-judiciary",
     "CRUD": "CRU"
+  },
+  {
```

Comment:
I would suggest making this available in all states. Court staff do cannot see a case until it's in submitted onwards anyway

Reply:
cool will do this :) i've wrote this down as a reminder court stall can't see until submitted. haha!!